src/main/java/com/solr/clientwrapper/domain/service/DataIngectionService.java

```diff
@@ -0,0 +1,64 @@
+package com.solr.clientwrapper.domain.service;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.solr.clientwrapper.domain.port.api.DataIngectionServicePort;
+
+@Service
+@Transactional
+public class DataIngectionService  implements DataIngectionServicePort{
+	
+	private final Logger log = LoggerFactory.getLogger(DataIngectionService.class);
+	SolrClient solrClient=new HttpSolrClient.Builder("http://localhost:8983/solr/docparse").build();
```

Comment:
The URL should be taken from the config file. Please remove this hardcoded URL

Reply:
Ravikiran's code. I will see if his updated version takes care of these things and will update mine then